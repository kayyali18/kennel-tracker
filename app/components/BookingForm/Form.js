import React from 'react'

// import Form 1
import Form1 from './Form1'

function Form() {
  let form = 0

  form = (
    <Form1>
      <input placeholder="Name" type="text" />
      <input placeholder="Address" type="text" />
      <input placeholder="Cellphone" type="number" />
      <input placeholder="Homephone" type="number" />
      <input placeholder="E-mail" type="email" />
      <input placeholder="Emergency Contact Name" type="text" />
      <input placeholder="Emergency Contact #" type="number" />
    </Form1>
  )

  return <div>{form}</div>
}

export default Form
