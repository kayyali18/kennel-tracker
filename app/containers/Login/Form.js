import styled from 'styled-components'

const Form = styled.form`
  display: flex;
  width: 450px;
  align-items: center;
  justify-content: center;
  margin: 0 auto;
  background: #fff;
`
export default Form
