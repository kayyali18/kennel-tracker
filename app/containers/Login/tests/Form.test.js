import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Form from '../Form'

describe('<Form />', () => {
  it('should match snapshot', () => {
    const renderedComponent = renderer.create(<Form />).toJSON()
    expect(renderedComponent).toMatchSnapshot()
  })

  it('should render a form', () => {
    const renderedComponent = renderer.create(<Form />).toJSON()
    expect(renderedComponent.type).toEqual('form')
  })
})
