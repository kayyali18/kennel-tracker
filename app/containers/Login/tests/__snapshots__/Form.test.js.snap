// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> should match snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 700px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: 0 auto;
}

<form
  className="c0"
/>
`;
